_data/actions/410-bgee.yml

```diff
@@ -4,4 +4,11 @@ name: RandomWalkTime
 params:
   - name: Time
     type: i
-unknown: true
+desc: |-
+  It is the same as [RandomWalk()](#85), where the time parameter represents how many times the action will be ticked before terminating (Actions are normally ticked once per second at 30FPS).
```

Comment:
> Actions are normally ticked once per second at 30FPS
This is a bit fishy. The "FPS" in the setting defaults to 30, twice the number of AI updates per second. So there are 15 ticks per second. Script updates do get staggered, but this description is a bit misleading.

I think it would be much clearer if we made the description talk about a count (how many times to run the main workload), not time as in timing.

Reply:
> > Actions are normally ticked once per second at 30FPS
> This is a bit fishy. The "FPS" in the setting defaults to 30, twice the number of AI updates per second. So there are 15 ticks per second. Script updates do get staggered, but this description is a bit misleading.
> 
> I think it would be much clearer if we made the description talk about a count (how many times to run the main workload), not time as in timing.

What are you suggesting exactly?

Reply:
How about:
> where the time parameter represents a limit on how many times the action will result in walking. Turning or waiting does not count against this limit. Since it is random, there is no guarantee movement will be immediate.
Even there the second sentence is probably not needed.

Then also the note becomes redundant and can be removed.

Reply:
This comment for the two new *time actions is the last thing remaining for the merge. 👍🏾

Reply:
Keep in mind that at least on EE games, FPS can be [increased](https://github.com/Argent77/A7-HiddenGameplayOptions/blob/784e147d59256f5300ed5a56b12163038a5a388e/HiddenGameplayOptions/setup-HiddenGameplayOptions.tp2#L327) (that's why I specified _Actions are normally ticked once per second at 30FPS_).

The second sentence is just to give an example.
Guess I could specify that if everything goes well (the creature doesn't go off-screen), `RandomWalkTime(1)` will terminate after one round...

Reply:
> This comment for the two new *time actions is the last thing remaining for the merge. 👍🏾

I'd like to include some other `PRO` flags before merging...

Reply:
AI "FPS" could be changed in the originals as well, but this action doesn't really care about it. Sure, the side effects of the executed Wait would be different, but that goes for all timings anywhere.

PRO: no problem with a separate PR — this one is a monster already.

Reply:
> AI "FPS" could be changed in the originals as well, but this action doesn't really care about it. Sure, the side effects of the executed Wait would be different, but that goes for all timings anywhere.

OK, fine.
I changed description using your words above.